crates/rome_control_flow/src/lib.rs

```diff
@@ -29,6 +29,16 @@ impl<L: Language> ControlFlowGraph<L> {
             node,
         }
     }
+
+    pub fn get(&self, id: BlockId) -> &BasicBlock<L> {
+        // SAFETY: safe conversion because a block id corresponds to its index in `self.blocks`.
+        let block_index = id.index() as usize;
+        &self.blocks[block_index]
+    }
+
+    pub fn block_ids(&self) -> impl Iterator<Item = (BlockId, &BasicBlock<L>)> {
+        (0..).map(|index| BlockId { index }).zip(self.blocks.iter())
```

Comment:
What's this function for?  I suppose the documentation will help

Reply:
I don't like the function however we need this in `noUnreacahble`

Reply:
I added docs. Does it help?

Reply:
Not really :( 

Also I don't understand why we do `(0..).map(|index| BlockId { index })`

Reply:
The function returns pairs `(block_identifier, block)`. The block identifier corresponds to its index in `self.blocks`.
An alternative implementation could be:

```rs
self.block.iter().enumerate().map(|(index, block)| (BlockId { index }, block))
```

Reply:
In fact, I used the first version because it is more concise and avoids a cast: I forgot about the fact that `enumerate` emits `usize` and `index` is a `u32`.